### PR TITLE
Add the atom-maker lib

### PR DESCRIPTION
**Pull request by marialivia16**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -4,4 +4,20 @@ play.application.loader=AppLoader
 
 play.crypto.secret="Replace me, please!"
 
-panda.system="atom-workshop"
\ No newline at end of file
+panda.domain="local.dev-gutools.co.uk"
+
+panda.authCallback="https://atomworkshop.local.dev-gutools.co.uk/oauthCallback"
```
Comment (philmcmahon) [inline conf/application.conf]: this and panda.domain shouldn't be here any more - should be moved to ~/.configuration-magic

File: app/data/AtomDataStore.scala [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient
+import com.gu.atom.data.PreviewDynamoDataStore
+import com.gu.contentatom.thrift._
+import com.gu.contentatom.thrift.atom.media.MediaAtom
+import com.gu.scanamo.scrooge.ScroogeDynamoFormat._
+import com.gu.scanamo.DynamoFormat
+import DynamoFormat._
+import cats.syntax.either._
+
+class PreviewAtomDataStore(dynamo: AmazonDynamoDBClient, tableName: String) {
```
Comment (philmcmahon) [inline app/data/AtomDataStore.scala]: This could possibly become `AtomDataStores`, with a `val previewStore` and `val publishedStore` inside it

Comment (marialivia16) [inline app/data/AtomDataStore.scala]: Good idea! I'm updating this now 😄

**Comment by philmcmahon** [comment]
👍

**Pull request merged**